mol_generator.py

```diff
@@ -0,0 +1,180 @@
+"""
+Using the schrodinger Python modules, read a template .mae file to
+generate the coordgen template C++ files.
+"""
+
+import sys
+import os
+import textwrap
+import argparse
+import subprocess
+
+from schrodinger import structure
+
+NAME = 'CoordgenTemplates'
+
+# String that creates a molecule and adds it to a container
+# of sketcherMinimizerMolecule* at index i.
+#
+# C++ programmers: `{` and `}` are escaped by duplication, so
+# `{{` means `{`.
+_MOLECULE = """
+{{
+    auto molecule = new sketcherMinimizerMolecule();
+    std::array<std::tuple<int, float, float>, {atom_total}> atoms = {{{{
```

Comment:
Given that this is just a template in a code-generation function, I don't think it's critical, but the duplication of the name `atoms` here makes my head hurt.
I'd rename this variable to `atomVect`